documentation/src/main/asciidoc/topics/code_examples/EvictionPassivation.java

```diff
@@ -0,0 +1,3 @@
+ConfigurationBuilder builder = new ConfigurationBuilder();
+builder.memory().maxCount(100);
+builder.persistence().passivation(true);//Persistent storage configuration
```

Comment:
the lack of space before the comment irks me :)

Reply:
fixed